### PR TITLE
WIP Fix Dataframes to groupedbar

### PR DESCRIPTION
**Pull request by mkborregaard**

### CODE REVIEW

File: src/bar.jl [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
+    Plots.extractGroupArgs(collect(g.args[1][group]))
+end
 
 @recipe function f(g::GroupedBar)
+    if isa(g.args[1], DataFrame)
+        d, g.args = processDF(d, g.args[1], g.args[2:end]...)
+    end
     x, y = grouped_xy(g.args...)
+    @show x
+    @show y
```
Comment (pkofod) [inline src/bar.jl]: are these just "temp" development values?

Comment (mkborregaard) [inline src/bar.jl]: Yes, this PR currently stinks. The WIP label doesn't lie. I was going to revamp the groupedbar recipe to accept a DataFrame based loosely on the suggestion in #44 but haven't made progress for months.

Comment (pkofod) [inline src/bar.jl]: Alright, was just browsing through!

**Comment by mkborregaard** [comment]
No longer necessary after https://github.com/JuliaPlots/StatPlots.jl/pull/83

**Pull request closed without merging**